mkdocs.yml

```diff
@@ -62,3 +68,30 @@ plugins:
   - search:
       separator: '[\s\-,:!=\[\]()"/]+|(?!\b)(?=[A-Z][a-z])'
 
+#Page tree
+nav:
+  - Home:
+    - index.md
+    - CONTRIBUTING.md
+  - Getting started:
+    - Installation: install.md
+    # - Configuration:
+    #   - configuration/index.md
+    #   - Profiles: configuration/profiles.md
+    #   - Theming: configuration/theming.md
+    #   - Images (Advanced) : configuration/advanced/images.md
+    #   - Mouse (Advanced) : configuration/advanced/images.md
+    #   - Misc (Advanced) : configuration/advanced/mouse.md
+    #   - Rendered (Advanced) : configuration/advanced/rendered.md
+  - Features:
+    - List of features: features.md
+    - Font liagatures: demo/font-ligatures.md
+    - Images: demo/images.md
+    - Input modes: demo/input-modes.md
+    - Line marks: demo/line-marks.md
+    - Statusline: demo/statusline.md
+    - Input modes: input-modes.md
```

Comment:
"Input modes" is listed twice.

I think the second one (line 93 currently) should be part of the actual user documentation, whereas stuff in the demo/ directory is more like a pitch to the user, of the main eye-catching features we're offering.

Reply:
Only one is left